src/ResponseHeaders.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+namespace Jasny\HttpMessage;
+
+/**
+ * ServerRequest header methods
+ */
+class ResponseHeaders extends Headers
+{
+    
+    /**
+     * HTTP headers
+     *
+     * @var bool
+     */
+    protected $isStale;
+
+    /**
+     * Create header array from resived array in the Header дшые
+     *
+     * @param array $incomingArray
+     * @response Header class object
+     */
+    public function __construct($incomingArray = [])
+    {
+        $this->headers = [];
+    
+        foreach ($incomingArray as $name => $values) {
+            $this->assertHeaderName($name);
+            $this->assertHeaderValue($values);
+            header($name . ': ' . implode(', ', (array)$values));
+        }
+    }
+    
+    /**
+     * For the current object simple chek  if current object 
+     * can be used for set or modify or remove headers
+     *
+     * @throws \InvalidArgumentException
+     */
+    protected function assertStale()
+    {
+        if ($this->isStale !== null) {
+            throw new \InvalidArgumentException("Can not change stale object");
+        }
+    }
+
+    /**
+     * Change current object to stale, move all headers to the variable
+     */
+    protected function setClassStale()
```

Comment:
Rename to `turnStale`, so it's the same as in [`ServerRequest\GlobalEnvironment`](https://github.com/jasny/http-message/blob/master/src/ServerRequest/GlobalEnvironment.php#L95).